docs/spec/bank/WIP_keeper.md

```diff
@@ -1,25 +0,0 @@
-WORK IN PROGRESS 
-See PR comments here https://github.com/cosmos/cosmos-sdk/pull/2072
-
-# Keeper
-
-## Denom Metadata
```

Comment:
Don't we still want this metadata store?

Reply:
AFAIU This is moreso a spec for what we want at launch, not a spec for the ideal #pre-ibc

Reply:
This should be done prelaunch.  It makes the staking codebase much simpler.

Reply:
OK, agreed, I'll do that in this PR.